content/blog/2020-07-09-cloud-update/index.Rmarkdown

```diff
@@ -0,0 +1,128 @@
+
+
+## What is RStudio Cloud?
+
+If you are used to using the [RStudio IDE](https://rstudio.com/products/rstudio/download/) for data science, this setup should look familiar:
+
+```{r echo=FALSE, out.width="75%", fig.cap="IDE screenshot from Dr. Mine Çetinkaya-Rundel"}
+knitr::include_graphics("https://mine-cetinkaya-rundel.github.io/rstudio-cloud-webinar/img/rstudio-ide.png")
+```
+
+RStudio Cloud moves the IDE into your web browser instead:
+
+```{r echo=FALSE, out.width="75%", fig.cap="Cloud screenshot from Dr. Mine Çetinkaya-Rundel"}
+knitr::include_graphics("https://mine-cetinkaya-rundel.github.io/rstudio-cloud-webinar/img/rstudio-cloud.png")
+```
+
+This makes [RStudio Cloud](https://rstudio.cloud) a lightweight way for anyone to do, share, teach, and learn data science online. For educators, it simplifies the process of teaching R to your students or colleagues by letting the instructor focus on the content, not the infrastructure. Students learn directly within their web browsers, with nothing to install locally, and with no infrastructure for the instructor to maintain.
+
+RStudio Cloud is designed from the ground up to make data science teaching easier for instructors and students:
+
+-   Classes can be organized in workspaces
+-   Members can be assigned different roles: instructor, TA, student
+-   Projects can be public or private
+-   Projects can be made into assignments
+-   Students can make copies of projects created by the instructor, with the necessary environment automatically replicated.
+-   Instructor can peek into and edit student projects
+
+In addition to teaching data science, RStudio Cloud is a great platform for doing data science as well. The ability to share projects makes it easy for researchers from different groups or institutions to collaborate, and is useful for any data science team that wants to build and share data science work, without having to maintain their own IT infrastructure.
+
+To learn more:
+
+-   Visit <https://rstudio.com/products/cloud>,
+-   Check out our recent webinar on [Teaching R Online with RStudio Cloud](https://rstudio.com/resources/webinars/teaching-r-online-with-rstudio-cloud/),
+-   Sign up at <https://rstudio.cloud/> and get started for free.
+
+## What's changing with RStudio Cloud?
+
+After an extensive alpha/beta period, RStudio Cloud will be generally available for use and purchase in the fall. (Until then, you can still sign up for the Cloud Free plan - the usage limits are currently not enforced). While RStudio Cloud will always offer a free plan for individuals, beginning August 3, 2020, we will also offer paid premium plans for individuals, instructors, students and organizations. Learn more on our [Plans & Pricing page](https://rstudio.cloud/plans/).
+
+## Why are we making this change?
```

Comment:
I think you could drop this heading — it's an obvious segue from the previous paragraph.